Segments/Kesmai.mapproj

```diff
@@ -140777,7 +140777,14 @@
         MaxHealth = 50, Health = 50,
         
         Experience = 50,
+        
+        Immunity = CreatureImmunity.Piercing | CreatureImmunity.Slashing | CreatureImmunity.Bashing |
```

Comment:
I think you just need IsInvulnerable = true, 

this would take care of all scenarios.